modules/generator/processor/servicegraphs/config.go

```diff
@@ -62,4 +67,10 @@ func (cfg *Config) RegisterFlagsAndApplyDefaults(string, *flag.FlagSet) {
 	cfg.PeerAttributes = peerAttr
 
 	cfg.EnableMessagingSystemLatencyHistogram = false
+
+	cfg.DatabaseNameAttributes = []string{
+		string(semconvnew.DBNamespaceKey),
```

Comment:
i nearly commented that these are out of order with the previous checks, but these are in fact correct. the previous logic is just awfully convoluted. nice catch